quad.go

```diff
@@ -0,0 +1,159 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package quad provides numeric integration of single variable functions by quadrature.
+//
+package quad
+
+import (
+	"math"
+	"sync"
+)
+
+// FixedLocationer computes a set of quadrature locations and weights and stores
```

Comment:
No, I don't think it should be forced. I'll change the FixedLoctionSingle to be "bounds from min to max". If the quadrature rule wants to give points outside the bounds, that's its prerogative.